src/components/domain/modals/TaskCardCreateModal.tsx

```diff
@@ -0,0 +1,246 @@
+                dayClassName={() => styles.day} // 하루하루 날짜 스타일
+                locale={ko}
+              />
+            </div>
+          </div>
+          <div className="flex flex-col pb-[3.2rem] gap-[0.8rem]">
+            <label className="text-2lg-medium text-[var(--black-000000)]">
+              태그
+            </label>
+            <div className="flex flex-wrap w-full min-h-[5rem] px-[1.6rem] py-[1rem] border border-[var(--gray-D9D9D9)] rounded-lg gap-[1rem] focus-within:border-[var(--violet-5534DhA)]">
+              {tags.map((tag, idx) => (
+                <Tag key={idx} label={tag.label} color={tag.color} />
+              ))}
+
+              <input
+                type="text"
+                value={inputValue}
+                onChange={(e) => setInputValue(e.target.value)}
+                onKeyDown={handleTagKeyDown}
+                placeholder={tags.length === 0 ? '입력 후 Enter' : ''}
+                className="flex-1 min-w-[10rem] bg-transparent text-lg-regular text-[var(--black-333236)] placeholder-[var(--gray-9FA6B2)] outline-none"
+              />
+            </div>
+          </div>
+          <div className="flex flex-col pb-[3.2rem] gap-[0.8rem]">
+            <label className="text-2lg-medium text-[var(--black-000000)]">
+              이미지
+            </label>
+            <div
+              onClick={handleImageClick}
+              className="w-[7.6rem] h-[7.6rem] relative flex items-center justify-center rounded-[0.6rem] bg-[#F5F5F5] cursor-pointer overflow-hidden"
+            >
+              {preview ? (
+                <Image
+                  src={preview}
+                  alt="업로드된 이미지"
+                  fill
+                  className="rounded-[0.6rem] object-cover"
+                />
+              ) : (
+                <Image
+                  src="/assets/icon/add.svg"
+                  alt="이미지 추가"
+                  width={17}
+                  height={17}
+                />
+              )}
+            </div>
+            {/* 숨겨진 파일 인풋 */}
+            <input
+              className=""
+              ref={inputRef}
+              type="file"
+              accept="image/*"
+              onChange={handleImageChange}
+              style={{ display: 'none' }}
+            />
+          </div>
+          <div className="w-full flex justify-center items-center gap-[0.8rem]">
+            <CommonButton
+              variant="secondary"
+              padding="1.4rem 11.4rem"
+              isActive={true}
+              className="w-full h-[5.4rem] rounded-lg text-[var(--gray-787486)] text-lg-medium border border-[var(--gray-D9D9D9)]"
+            >
+              취소
+            </CommonButton>
+            <CommonButton
+              variant="primary"
+              padding="1.4rem 11.4rem"
+              isActive={true}
+              className="w-full h-[5.4rem] bg-[var(--violet-5534DhA)] text-[var(--white-FFFFFF)] text-lg-semibold"
```

Comment:
text-lg와 font-semibold를 따로 넣어야할것 같아요.
- https://tailwindcss.com/docs/font-size
- https://tailwindcss.com/docs/font-weight

```suggestion
              className="w-full h-[5.4rem] bg-[var(--violet-5534DhA)] text-[var(--white-FFFFFF)] text-lg font-semibold"
```

Reply:
이 부분 역시 globals.css에서 선언한 것을 가져다 사용했습니다!